src/FileJobClientBrowser.ts

```diff
@@ -0,0 +1,116 @@
+import axios from "axios";
+
+import { JobClient } from "./JobClient";
+import { Logger } from "./Logger";
+
+import type {
+  ClassInitiator,
+  SubmitJobFileParams,
+  SubmitJobResponse,
+} from "./types";
+
+export class FileJobClient {
+  logger: Logger;
+  readonly baseUrl: string;
+  readonly headers: {
+    Authorization: string;
+  };
+  jobClient: JobClient;
+
+  /**
+   * Creates a JobClient
+   * @param {Object} config object
+   * @param {string} config.url - base url of modzy api (i.e.: https://app.modzy.com/api)
+   * @param {string} config.apiKey - user's API key
+   */
+  constructor({
+    url = "https://app.modzy.com/api",
```

Comment:
I presume this is overridable, but can we maybe defined this as a constant somewhere and refer to the constant rather than copy/paste the same string everywhere?

Also, I think the URL should be `https://app.modzy.com` and _not_ include the `/api` by default. That should be part of the path when you call each function (e.g. `/api/jobs` rather than `/jobs`). 

Thoughts?

Reply:
Raul had it where the `/api` was included in the url, but it honestly confused me multiple times when editing this stuff. I kept it with the mindset of minimizing api changes, but that's dumb given how I ended up blowing up everything else. I'll remove it and make us all happy

Reply:
And yes, overridable like ...
```javascript
// app.modzy.com
const modzyClient = new ModzyClient({
  apiKey: "xxxxxxxxxxxxx.xxxxxxxxxxxxx",
});

// or for private Modzy instances
const modzyClient = new ModzyClient({
  apiKey: "xxxxxxxxxxxxx.xxxxxxxxxxxxx",
  url: "https://modzy.yourdomain.com",
});
```